modules/publicize/publicize.php

```diff
@@ -502,6 +513,13 @@ function get_publicize_conns_test_results() {
 					}
 				}
 
+				// LinkedIn needs reauthentication to be compatible with v2 of their API
+				if ( 'linkedin' === $service_name && $this->is_invalid_linkedin_connection( $connection ) ) {
+					$connection_test_passed = 'must_reauth';
+					$user_can_refresh = false;
+					$connection_test_message = esc_html__( 'Your LinkedIn connection needs to be reauthenticated to continue working – head to Sharing to take care of it.' );
```

Comment:
Could you add the `jetpack` textdomain here and for the other strings?

Reply:
I didn't add it because I saw other strings in these files and they don't have the `jetpack` domain. I thought that was weird but didn't add it thinking that maybe it's something that works with fusion.

Reply:
It may be worth taking the opportunity to add the missing textdomains for the other strings at the same time?

Reply:
Fixed in 9d65bd434.